nifi-nar-bundles/nifi-snmp-bundle/nifi-snmp-processors/src/test/java/org/apache/nifi/snmp/helper/configurations/SNMPV1Configurations.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.nifi.snmp.helper.configurations;
+
+import org.apache.nifi.snmp.configuration.SNMPConfiguration;
+import org.snmp4j.mp.SnmpConstants;
+
+public class SNMPV1Configurations implements SNMPConfigurations {
```

Comment:
I think, there is too much util class for testing. Would not it be possible to move the code into the relevant tests? My reasoning:
- The tests become too fragmented and because of this: hard to read
- As I see, most of the calls are used at one place, which does not reason extracting in a test
- The SNMPConfigurations abstraction is implemented in such a ways that V1... etc are used in the respected test file (V1... etc) only, thus the abstraction is not really provides benefit

Reply:
The TestRunnerFactories use the SNMPConfigurationFsctories to create test runners. I think it would increase the test complexity and duplicated code too much. I removed the version based factory providers.